index.html

```diff
@@ -1475,6 +1475,114 @@ <h2>Relationship to Linked Data</h2>
+This section lists cryptographic hash values that might change during the
+Candidate Recommendation phase based on implementer feedback that requires
+the referenced files to be modified.
+        </p>
+        <p>
+Implementations that perform JSON-LD processing MUST treat the following
+JSON-LD context URLs as already resolved, where the resolved document matches
+the corresponding hash values below:
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>URL and Media Type</th>
+              <th>Content</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td style="white-space: nowrap;">
+https://www.w3.org/ns/data-integrity/v1<br>
+application/ld+json
+              </td>
+              <td>
+sha256: v/POI0jhSjPansxhJAP1fwepCBZ2HK77fRZfCCyBDs0=<br><br>
+sha3-512: Sg1PLFxKyEYQns9Zr0BoYXtFeDNfrHUDNMkyq4QEWv<wbr>wGIaX1v5xovnCG+dfceZEzr7BhBjm396noZF1HEeCM8g==
+              </td>
+            </tr>
+            <tr>
+              <td style="white-space: nowrap;">
+https://www.w3.org/ns/multikey/v1<br>
+application/ld+json
+              </td>
+              <td>
+sha256: SA+P0RxXl8ZH6xdPcmX71crzeGkCYMPfYjdVT46SF0o=<br><br>
+sha3-512: BgQBN00t6vj1T4uYt5SwdPOhe2BVNY11UEvkVo/rbBv<wbr>fBd+Z09isxhGhf2HkVqMT
+SANah1u7rLk6Uw6GeQGH1w==
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p>
+Implementations that perform RDF vocabulary processing MUST 1) be implemented
+such that the vocabulary URLs used in the JSON-LD contexts listed above
+resolve to the files listed below, and 2) match the listed cryptographic
+hashes (if specified). The vocabulary content at the URLs listed below are
+normative.
+        </p>
+
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>URL and Media Type</th>
+              <th>Content</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+https://w3id.org/security#<br>
+text/html
+              </td>
+              <td>
+https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.html
+              </td>
+            </tr>
+            <tr>
+              <td>
+https://w3id.org/security#<br>
+application/ld+json
+              </td>
+              <td>
+https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.jsonld<br><br>
+sha256: snzV1JA8H7YB4AAQIH6qLzv48ZOhJ3oCl/ewvf0EBMg=<br><br>
+sha3-512: 7QUmDvF24f0/J0I7ZYpZrtlGIHcsjGXGfjBseg5e3j<wbr>xDZlTqd4P7tklHdKn3fwmT
+yKO1kwi0jv9/DLNjWbMrHA==
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <p class="issue" title="w3c.github.io links expected to change">
+The URLs listed above that start with
+`https://w3c.github.io/vc-data-integrity/vocab/security/` are expected to change
+to `https://www.w3.org/ns/security/` or an equally normative and archived
+location under W3C control.
+        </p>
+
```

Comment:
I am a bit puzzled by this remark, and I am not sure where we stand right now. Do we plan to:

- Set up `https://www.w3.org/ns/security/vocabulary.{jsonld,ttl,html}, but continue using `w3id.org`? Or
- Same as above, but removing `w3id.org` from the equation?

In both cases the `/ns/security` is good to have because it may serve as a final resting place for the vocabulary files once the development is over, giving it the same level of 'rank' as `/TR` for the HTML versions. The question is the future role and usage of `w3id.org`

(Obviously, this remark is not a show-stopper for the PR, although the note may also say that the `w3id.org` usage may also change if that is the intention.)

Reply:
> (Obviously, this remark is not a show-stopper for the PR, although the note may also say that the w3id.org usage may also change if that is the intention.)

That's not the intention. The intention was to say that "the official version is under W3C control and any URL starting with `https://w3id.org/security` MUST be treated as resolving to the URL hosted by W3C". That is how we can ensure that w3id.org URLs can be used for redirection as things are being incubated, and then standardized without creating disruption to already deployed systems. It's a design pattern that's meant to not create disruption as things are standardized vs. the disruption we'd cause by changing these URLs now.

Removing w3id.org from the equation at this point would break production deployments and already deployed global standards from other standards bodies, such as 1EdTech (CLRv2 and OpenBadges), and Conexxus (Age Verification standard). See these JSON-LD context files (used by finalized standards):

https://purl.imsglobal.org/spec/clr/v1p0/context/clr_v1p0.jsonld (note use of w3id.org URLs and the security vocabularies)
https://convenience-org.github.io/age-verification-context/contexts/age-v1.jsonld (note use of w3id.org URLs and the security vocabularies)

We might be able to do it for terms that are not in production yet (like some of the status list properties), but `proof`, `multibase`, `digestMultibase`,  and `publicKeyMultibase` are already in production (and in use by other standards organizations), so changing them at this point would be disruptive.

Reply:
@msporny, understood. However, the `/ns` namespace is usually used as the public, final URL of a vocabulary. I.e., and in spite of my comment above (I was wrong), I do not think option (1) is a good way forward. If the vocabulary URL continues to be `w3id.org` then the physical storage of the vocabularies themselves is not really of importance: we can leave them on github while the development is happening, and we can store them on the W3C date space later to provide long term security. Put it another way, `w3id.org` and `www.w3.org/ns/security/` are mutually exclusive, imho.